go.mod

```diff
@@ -155,3 +155,5 @@ replace google.golang.org/grpc => google.golang.org/grpc v1.33.2
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
 replace github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
+
+replace github.com/cosmos/cosmos-sdk => github.com/yihuang/cosmos-sdk v0.44.6-0.20211213082904-42a724c145bd
```

Comment:
TODO: update after https://github.com/cosmos/cosmos-sdk/pull/10725 merged and backported to 0.44.x.

Reply:
isn't this a consensus change?

Reply:
UPDATE: it's merged in release/v0.44.x, but not tagged yet.